CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to Contribute
+
+First of all, thanks for considering contributing! ✨ It's people like you that make open-source tools awesome. 💖
+
+At Stoplight, we want contributing to any of our tools to be an enjoyable and educational experience for everyone. Contributions go beyond commits in pull requests. We are excited to receive contributions in the form of feature ideas, pull requests, triaging issues, reviewing pull requests, implementations in your own projects, blog posts, talks referencing the project, tweets, and much more.
+
+## Stoplight Community Code of Conduct
+
+The Stoplight Community is dedicated to providing a safe, inclusive, welcoming, and harassment-free space and experience for all community participants, regardless of gender identity and expression, sexual orientation, disability, physical appearance, socioeconomic status, body size, ethnicity, nationality, level of experience, age, religion (or lack thereof), or other identity markers. 
+
+Our Code of Conduct exists because of that dedication, and we do not tolerate harassment in any form. See our reporting guidelines [here](https://github.com/stoplightio/code-of-conduct/blob/master/incident-reporting.md). Our full Code of Conduct can be found at this [link](https://github.com/stoplightio/code-of-conduct/blob/master/long-form-code-of-conduct.md#long-form-code-of-conduct).
+
+## Development
+
+Yarn is a package manager for your code, similar to npm. While you can use npm to use json-ref-resolver in your own project, we use yarn for development.
+
+1. [Fork](https://help.github.com/articles/fork-a-repo/) this repository to your own GitHub account and then [clone](https://help.github.com/articles/cloning-a-repository/) it to your computer.
+2. Install yarn: `npm install -g yarn`
```

Comment:
I think the yarn maintainers themselves discourage this method and push for os specific solutions, however this will just work.

Reply:
Ahh, didn't know that. I'm just copying and pasting from Spectral for consistency but I'll sort this all out sometime if it causes trouble.